build.savant

```diff
@@ -1,20 +1,8 @@
 /*
- * Copyright (c) 2018-2023, Inversoft, All Rights Reserved
- *
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *   http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing,
- * software distributed under the License is distributed on an
- * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
- * either express or implied. See the License for the specific
- * language governing permissions and limitations under the License.
+ * Copyright (c) 2024, Inversoft Inc., All Rights Reserved
```

Comment:
Looks like you broke the copyright config for this repo? This repo is licensed under Apache v2.

Reply:
AFAIK, the copyright config in IJ never accounted for this. Just fixed this file and fixed the rules in https://github.com/CleanSpeak/cleanspeak-ij/pull/2